### PR TITLE
changelog: Matomo 5.0.1

### PR DESCRIPTION
**Pull request by davlgd**
## Checklist

- [ ] My PR is related to an opened issue : #

**⚠️ If changes affect GitHub Action files**

- [x] I have added tests to cover my changes : [link](url)

## Pull request type

Please check the type of change your PR introduces:

- [x] Documentation content changes
- [ ] Bugfix on the site
- [ ] Build related changes
- [ ] Other (please describe):
      
## Description
Mention Matomo 5.0.1 in changelog

### CODE REVIEW

File: content/changelog/2024-01-22-matomo-5.0.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Matomo 5.0.1 is available
```
Comment (miton18) [inline content/changelog/2024-01-22-matomo-5.0.1.md:2]:
```suggestion
title: Matomo v5.0.1 is available
```

File: content/changelog/2024-01-22-matomo-5.0.1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - name: Rémi Collignon-Ducret
+    link: https://github.com/miton18
+    image: https://github.com/miton18.png?size=40
+description: You can now deploy Matomo 5.0.1 on Clever Cloud
+excludeSearch: true
+---
+
+Matomo 5.0 was released mid-december, version 5.0.1 is now available. Since then, we've worked on the support of this new branch on Clever Cloud. It's done! You can deploy this latest version right now from our [Console](https://console.clever-cloud.com) or [Clever Tools](https://github.com/CleverCloud/clever-tools). 
+
+Existing customers instances are already up-to-date.
```
Comment (miton18) [inline content/changelog/2024-01-22-matomo-5.0.1.md:20]: From customer point of view, they ordered an addon
)
```suggestion
Existing customers addons are already up-to-date.
```

**Review by miton18 — DISMISSED** [review]

File: content/changelog/2024-01-22-matomo-5.0.1.md (hunk 2 above)
Comment (juliamrch) [reply content/changelog/2024-01-22-matomo-5.0.1.md:20]: The chosen syntax is `add-on` in the docs

File: content/changelog/2024-01-22-matomo-5.0.1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  - name: Rémi Collignon-Ducret
+    link: https://github.com/miton18
+    image: https://github.com/miton18.png?size=40
+description: You can now deploy Matomo 5.0.1 on Clever Cloud
+excludeSearch: true
+---
+
+[Matomo](https://matomo.org/) 5.0 was released mid-december, version 5.0.1 is now available. Since then, we've worked on the support of this new branch on Clever Cloud. It's done! You can deploy this latest version right now from our [Console](https://console.clever-cloud.com) or [Clever Tools](https://github.com/CleverCloud/clever-tools). 
+
+Existing customers instances are already up-to-date.
```
Comment (juliamrch) [inline content/changelog/2024-01-22-matomo-5.0.1.md:20]:
```suggestion
Existing customers add-ons are already up-to-date.
```

**Review by juliamrch — APPROVED** [review]

**Pull request merged**